hedera-mirror-rest/api/v1/openapi.yml

```
@@ -300,7 +300,26 @@ paths:
   /api/v1/contracts/call:
     post:
       summary: Invoke a smart contract
-      description: Returns a result from EVM execution such as cost-free execution of read-only smart contract queries, gas estimation, and transient simulation of read-write operations. If `estimate` field is set to true gas estimation is executed.
+      description: |
+
+        Returns a result from EVM execution such as cost-free execution of read-only smart contract queries, gas estimation, and transient simulation of read-write operations. If `estimate` field is set to true gas estimation is executed.
+
+        **The table below defines the restrictions and support for the endpoint**
+
+        | Call type        | Operation Type                                               | Services Version | Mirror-Node Version | Support |
+        | -------------    | ------------------------------------------------------------ | ---------------- | ------------------- | ------- |
+        | eth_call         | static contract state reads for non precompile functions     | 0.36             | 0.78                | Y       |
```

Should add a `+` to the end of all versions to indicate it's in that release and any subsequent.
```suggestion
        | eth_call         | static contract state reads for non precompile functions     | 0.36+             | 0.78+                | Y       |
```

---

Done